config.json

```diff
@@ -1177,6 +1177,18 @@
       "unlocked_by": "luhn",
       "uuid": "37821140-c0d0-4da8-8f50-47356705a615"
     },
+    {
+      "core": false,
+      "difficulty": 4,
+      "slug": "proverb",
+      "topics": [
+        "arrays",
+        "loops",
+        "strings"
+      ],
+      "unlocked_by": "two-fer",
```

Comment:
Good job here @sunilgopinath the only suggestion I would have is for the exercise unlock, seems like it falls into the general category of repeat a string with slight positional variations, which sits nicely in this subtree under [twelve-days](../tree/master/exercises/twelve-days):

```text
├─ twelve-days
│  ├─ house
│  ├─ food-chain
│  └─ beer-song
│
```

We can merge this as is and patch that change later. It is related to the next-gen exercism which is a later concern. I imagine we may revisit all the ordering before that anyhow.

Reply:
That makes sense. I can update the ordering on another PR. I used the java repo as an example as I wasn't sure what to put there.

Reply:
Yeah it can be tricky and sort of language dependent, your approach to guesstimating is as good as any :smile:.